community/sway/etc/sway/config.d/99-autostart-applications.conf

```
@@ -12,7 +12,7 @@ exec {
     '[ -x "$(command -v wlsunset)" ] && /usr/share/sway/scripts/sunset.sh'
     '[ -x "$(command -v autotiling)" ] && autotiling'
     '[ -x "$(command -v sworkstyle)" ] && sworkstyle &> /tmp/sworkstyle.log'
-}
+    $term_server
```

what is going on here?
```suggestion
    $term_server
}
```

---

It is to launch the $term_server command defined in /etc/sway/definitions. Still not sure what's the most correct way to do this, probably can be rewritten to static `[ -x "$(command -v foot)" ] && foot --server`, if that would be better in your opinion, I can do this, now I realized it's over-engineering to add it. If a user will change to other client-sever terminal, $term_float shall be redefined anyway and a new server process has to be added to the ~/.config/sway/config.d/00-user.conf (or by changing $term_server, in the current variant). I'll change it I think

---

sorry. updated and makes sense.  with whats going on I meant the '}' being removed